extensions/node-version-manager/src/utils/shell.ts

```diff
@@ -0,0 +1,43 @@
+import { spawnSync } from 'node:child_process';
+import { env } from './common';
+
+const run = <T extends object | string>(fullCommand: string, parseOutput?: (op: string) => T): T => {
```

Comment:
Please update nvm manager to use this util, just to not duplicate code.

Reply:
Hi, sorry I prefer not to touch existing code as I have no time to refactor

Reply:
Ok, I think I can accept it, and then try to refactor it.

Reply:
thanks